src/App.js

```diff
@@ -1,5 +1,18 @@
+import LottoController from './Controller/index.js';
+
 class App {
-  play() {}
+  #controller;
+
+  constructor() {
+    this.#controller = new LottoController();
```

Comment:
철원님이 저번에 말씀해주셨던 부분이 있었는데,
```suggestion
    new LottoController();
```
해당 부분을 이렇게 사용하는건 어떻게 생각하시나요?